modules/serverless-nodejs-context-object-reference.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-functions-reference-guide.adoc
+
+[id="serverless-nodejs-context-object-reference_{context}"]
+= Node.js context object reference
+
+The `context` object has several properties that can be accessed by the function developer.
+
+[id="serverless-nodejs-context-object-reference-log_{context}"]
+== log
+
+Provides a logging object that can be used to write output to the cluster logs. The log adheres to the link:https://getpino.io/#/docs/api[Pino logging API].
+
+.Example log
+[source,js]
```

Comment:
One more `[source,js]` -> `[source,javascript]` 👍

Reply:
Oops, thank you!